Xlib/protocol/display.py

```diff
@@ -42,12 +42,41 @@
 from . import event
 
 if PY3:
-    def buffer(object, offset=None, size=None):
+
+    class bytesview(object):
+
+        def __init__(self, data, offset=None, size=None):
+            if offset is None:
+                offset = 0
+            if size is None:
+                size = len(data)-offset
+            if isinstance(data, bytes):
+                view = memoryview(data)
+            elif isinstance(data, bytesview):
+                view = data._view
+            else:
+                raise ValueError('unsupported type: %s' % type(data))
```

Comment:
OK, but we don't support 2.6 anymore. In fact, there's a extraneous `view = memoryview(data)` in the Python 2 code that made me realize that `memoryview` is also supported in Python 2, starting with 2.7, so we could further simplify the code by just using one `BytesView` class for all supported Python versions. What do you think?